test/e2e/variation_index.test.js

```diff
@@ -0,0 +1,126 @@
+const algoliasearch = require('algoliasearch');
+const sfcc = require('sfcc-ci');
+
+describe('Algolia Integration', () => {
+    beforeAll(async () => {
+        if (!process.env.CIRCLECI) {
+            console.log('Skipping Algolia Integration tests - not in CircleCI environment');
+            return;
+        }
+
+        const sfccAuthVars = ['SFCC_OAUTH_CLIENT_ID', 'SFCC_OAUTH_CLIENT_SECRET'];
+        sfccAuthVars.forEach(envVar => {
+            if (!process.env[envVar]) {
+                throw new Error(`Missing SFCC authentication variable: ${envVar}`);
+            }
+        });
+
+        // Authenticate with SFCC using the JavaScript API
+        try {
+            await new Promise((resolve, reject) => {
+                sfcc.auth.auth(
+                    process.env.SFCC_OAUTH_CLIENT_ID, 
+                    process.env.SFCC_OAUTH_CLIENT_SECRET,
+                    (err, token) => {
+                        if (err) {
+                            console.error('Failed to authenticate with SFCC:', err);
+                            reject(err);
+                            return;
+                        }
+                        process.env.ACCESS_TOKEN = token;
+                        console.log('Successfully authenticated with SFCC');
+                        resolve(token);
+                    }
+                );
+            });
+        } catch (error) {
+            console.error('Failed to authenticate with SFCC:', error);
+            throw error;
+        }
+
+        // Now verify other required environment variables
+        const requiredEnvVars = ['SANDBOX_HOST', 'ACCESS_TOKEN', 'ALGOLIA_APP_ID', 'ALGOLIA_API_KEY'];
+        requiredEnvVars.forEach(envVar => {
+            if (!process.env[envVar]) {
+                throw new Error(`Missing required environment variable: ${envVar}`);
+            }
+        });
+    });
+
+    let client;
+    let index;
+
+    beforeEach(() => {
+        // Skip setup if not in CircleCI
+        if (!process.env.CIRCLECI) return;
+
+        client = algoliasearch(process.env.ALGOLIA_APP_ID, process.env.ALGOLIA_API_KEY);
+        index = client.initIndex(process.env.ALGOLIA_INDEX_NAME || 'varx__products__en_US');
+    });
+
+    test('should search for a specific product', async () => {
+        if (!process.env.CIRCLECI) {
+            console.log('Test skipped - not in CircleCI environment');
```

Comment:
As I said above, this file doesn't seem to be executed. But even if it was executed, the tests would do nothing at all!
You chose to move away from CircleCI, you need to adapt your code accordingly...